### PR TITLE
Feat index.html: Add Content-Security-Policy tag

### PR DESCRIPTION
**Pull request by S2606**
**Changes proposed in this pull request**
- Add CSP meta tag for protection against XSS attacks

**Closes https://github.com/fossasia/loklak_search/issues/453**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
   <meta charset="utf-8">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'">
```
Comment (hemantjadon) [inline src/index.html]: Does it allow API calls? I am not sure?

**Comment by hemantjadon** [comment]
@S2606 any updates on this? Are you stuck somewhere?

**Comment by S2606** [comment]
@hemantjadon Can i pass this PR to someone else??

**Comment by hemantjadon** [comment]
@S2606 
Hey! Sure you can do that 😄 
But if you are stuck somewhere, we can help you to get around. We can work together on this.
You can take your time on this if you want. Just mention if you get stuck somewhere, we will definitely help 😄

**Comment by mariobehling** [comment]
Closing this due to inactivity.

**Pull request closed without merging**